diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -129,7 +129,7 @@ enum class Warnings(
     LONG_NUMERICAL_VALUES_SEPARATED(true, "3.14.2", "long numerical values should be separated with underscore"),
     MAGIC_NUMBER(false, "3.14.3", "avoid using magic numbers, instead define constants with clear names describing what the magic number means"),
     WRONG_DECLARATIONS_ORDER(true, "3.1.4", "declarations of constants and enum members should be sorted alphabetically"),
-    WRONG_MULTIPLE_MODIFIERS_ORDER(true, "3.14.1", "sequence of modifier-keywords is incorrect"),
+    WRONG_MULTIPLE_MODIFIERS_ORDER(true, "3.14.1", "sequence of modifier-keywords is incorrect. Value identifier supported in Kotlin 1.6"),
```

Comment:
I don't think it should be in a warning text. We have access to version set in `diktat-analysis.yaml` and can do some logic in the rule itself

Reply:
This is what I have asked. I need some mentions/tags of Kotlin 1.6 features

Reply:
We can use different list of keywords based on version from config. To aggregate we could add a marker annotation like `@MinimalLanguageVersion` for rules that are only used starting with a particular version or `@OptionalLanguageVersion` for rules that have part of their functionality available starting from a particular version (like this one).